### ISSUE TITLE
Generate archive

### ISSUE THREAD

**Issue by parkan** [mediachain/concat#133]
Probably just statements in a tarball. 1 statement = 1 "file" extent inside tarball?

**Comment by vyzo** [comment]
Obviously we also want the metadata objects, not just statements.

**Comment by vyzo** [comment]
Also, I don't think we need concat to dump the archive through an api at this point; we can generate it with a client script using the existing query/data api.

**Comment by parkan** [comment]
Yes, sorry I mean statements + data objects. I'm ok with using ndjson over protobuf.

**Comment by parkan** [comment]
Interesting, IPFS has some related work (though probably not mature):

https://github.com/ipfs/archive-format
